packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -53,7 +53,18 @@ export default class ReconnectionManager {
      * @memberof ReconnectionManager
     */
     this.webex = meeting.webex;
-    this.configure(meeting);
+    /**
+     * @instance
+     * @type {Meeting}
+     * @private
+     * @memberof ReconnectionManager
+     */
+    this.meeting = meeting;
```

Comment:
@adamweeks  would this not cause any issues when we change the actual meeting object ?. Also we will need a cleanup logic for meeting object on the parent object destroy !?

Reply:
Since it is an object, anything changed on the parent meeting object would be seen in the `this.meeting` reference. Not sure how you want to cleanup since the reconnection manager is a class instance that lives on the meeting object. When the meeting object is destroyed, so will the reconnection manager instance.

Reply:
I confirmed that this is created and mounted in only one place: `/meeting/index.js`. There shouldn't be any location that persistently stores `meeting.reconnectionManager` based on my investigation.